.github/scripts/binaryCompatibility.py

```diff
@@ -0,0 +1,78 @@
+#  Copyright Amazon.com Inc. or its affiliates.
+#  SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import json
+import os
+import xml.etree.ElementTree as ET
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Using xml.etree.ElementTree to parse untrusted XML data is known to be vulnerable to XML attacks. Replace xml.etree.ElementTree with the equivalent defusedxml package, or make sure defusedxml.defuse_stdlib() is called. https://bandit.readthedocs.io/en/latest/blacklists/blacklist_imports.html#b405-import-xml-etree